test/main/accounts/index.test.js

```diff
@@ -77,9 +90,7 @@ beforeEach(() => {
 
 afterEach(() => {
   Object.values(Accounts.accounts).forEach(account => {
-    Object.keys(account.requests).forEach(id => {
-      Accounts.removeRequest(account, id)
-    })
+    account.requests = {}
```

Comment:
doesn't this now leave the requests in the store where before it was clearing them?

Reply:
Possibly. Can't remember why I did this now. Could be because I'm not a huge fan of this half integration / half unit test style where we're using the Accounts API for setup & teardown, tried changing it and then went with the pragmatic route.

Reply:
Update: broke the tests.  I don't think we should be touching the store when these are meant to be units for accounts.

Reply:
Fixed by throwing some empty strings into the store for gas fee values. Still think we need a better solution for unit testing, specifically around the store, but that's a topic for another day...